CloudFormation/arkcase-foia-cft.yml

```diff
@@ -0,0 +1,112 @@
+---
+Parameters:
+  AMIImageId:
+    Type: String
+    Description: AMI ArkCase-FOIA
+    Default: ArkCase-FOIA
+  InstanceType:
+    Type: String
+    Description: Enter instance type for the EC2 Instance
+    Default: t2.xlarge
+    AllowedValues: 
+      - t2.xlarge
+      - t2.2xlarge
+  EC2InstanceName:
+    Type: String
+    Description: EC2 Instance Name
+    Default: ArkCase FOIA
+  VpcId:
+    Type: String
+    Description: Enter VPC id
+  SubnetId:
+    Type: String
+    Description: Enter Subnet in which to launch an EC2
+
+Mappings:
+  AMIMap: 
+    ArkCase-FOIA: 
+      imageId: "ami-07b3b6e1464caffd9"
+      name: "ArkCase-FOIA"
+
+Resources:
+  SSHSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId: !Ref VpcId
+      GroupDescription: Enable SSH access 
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+
+  ServerSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId: !Ref VpcId
+      GroupDescription: Allow connections from specified ranges 
+      SecurityGroupIngress:
+        - Description: http
+          IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
+        - Description: https
+          IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+  
+  ArkCaseFoiaKeyPair:
+    Type: AWS::EC2::KeyPair
+    Properties:
+      KeyName: arkcase-foia-cft-keypair
+
+  ArkcaseCEInstance:
```

Comment:
Should be `ArkCaseFOIAIntance` I guess, or else they can all just be called `ArkCaseInstance`.

Reply:
Right again, thank you.